Log Viewing/ARKLogTableViewController.m

```diff
@@ -503,28 +499,22 @@ + (NSDateFormatter *)sharedDateFormatter;
 
 #pragma mark - Initialization
 
-- (instancetype)initWithDate:(NSDate *)date;
+- (instancetype)init;
 {
-    self = [self init];
-    if (!self) {
-        return nil;
-    }
-    
-    _type = ARKLogTypeSeparator;
-    _creationDate = date;
-    
-    return self;
+    [NSException raise:NSInternalInconsistencyException format:@"Please use a valid initializer."];
+    return [self initWithDate:nil];
 }
 
-#pragma mark - Properties
+- (instancetype)initWithDate:(NSDate *)date;
+{
+    NSString *text = [[[self class] sharedDateFormatter] stringFromDate:date];
```

Comment:
This initializer is truly weird. Why would you want a message that just contained a date in both NSString and NSDate format?